index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <title>Portfolio Template 1</title>
+  </head>
+  <body>
+    <!-- start toolbar -->
+    <header>
+      <div class="logo_placeholder">
+        <a href="index.html" class="logo">My Logo</a>
+      </div>
+      <nav class="btn">
+        <i class="material-icons colored">menu</i>
+      </nav>
+    </header>
+    <!-- end toolbar -->
+
+    <!-- start main -->
+    <main class="shapes_bg">
+      <section class="headline">
+        <hgroup>
+          <h1>I’m Radmila</h1>
+          <h2>Glad to see you!</h2>
+        </hgroup>
+
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h3>Let's connect</h3>
```

Comment:
I suggest you make this a button with the necessary styling (hover, active, pressed ) as this is a call to action read this [Link](https://www.smashingmagazine.com/2009/10/call-to-action-buttons-examples-and-best-practices/) for more info